modules/gapi/test/cpu/gapi_imgproc_tests_fluid.cpp

```diff
@@ -106,7 +106,7 @@ INSTANTIATE_TEST_CASE_P(gaussBlurTestFluid, GaussianBlurTest,
                                 Values(-1),
                                 Values(IMGPROC_FLUID),
                                 Values(ToleranceFilter(1e-3f, 0.01).to_compare_obj()),
-                                Values(3))); // add kernel size=5 when implementation is ready
+                                Values(3, 5)));
```

Comment:
BTW can we extend kernel sizes of all of the kernels which utilize `run_sippfilter`? (not in this PR maybe)

Reply:
Unfortunately we can't extend kernel sizes of all kernels which utilize run_sepfilter() since 5x5 kernel size supports still wasn't added to all kernels.